src/files/ng-add-pug-loader.js

```diff
@@ -24,3 +24,28 @@ fs.readFile(commonCliConfig, (err, data) => {
     fs.close(file, () => {});
   });
 });
+
+/**
+ * Set's directTemplateLoading: false to allow custom pug template loader to work
+ * @see https://github.com/angular/angular-cli/issues/14534
+ */
+const typescriptCliConfig = '__TYPESCRIPT_CLI_CONFIG_PATH__';
+
+fs.readFile(typescriptCliConfig, (err, data) => {
+  if (err) { throw err; }
+
+  const typescriptText = data.toString();
+  
+  // update the setting
+  const output = typescriptText.replace('directTemplateLoading: true,', 'directTemplateLoading: false,');
```

Comment:
Please check if the file has this directive first, and return early. [Angular v6](https://github.com/angular/angular-cli/blob/v6.2.9/packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/typescript.ts#L72-L85) does not have it.

Reply:
Done.